platform/net/photonos_net_manager.go

```diff
@@ -0,0 +1,233 @@
+// Copyright © 2018 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache License 2.0
```

Comment:
Could you remove this copyright assignment from here? Copyrights are assigned to the CF Foundation on the NOTICE file at the root of the repo.

Reply:
Hi Luan,
May I know where should I add these CopyRights? Does this mean we should not write the VMware Copyright in this repo at all?